python_modules/libraries/dagster-dg/dagster_dg/cli/plus/deploy.py

```diff
@@ -167,7 +167,9 @@ def deploy_group(
     )
 
     cli_config = normalize_cli_config(global_options, click.get_current_context())
-    plus_config = DagsterPlusCliConfig.get()
+    plus_config = (
+        DagsterPlusCliConfig.get() if DagsterPlusCliConfig.exists() else DagsterPlusCliConfig()
+    )
     organization = _get_organization(organization, plus_config)
```

Comment:
(to inform my follow-up) does it seem right to you to fold in CLI config to `DagsterPlusCliConfig`? I imagine it should be the source of truth for handling merging CLI vars, env vars, config file?

Reply:
I think that's reasonable, yeah